Backend/app/email_router/email_service.py

```diff
@@ -0,0 +1,36 @@
+import email
+from fastapi import Depends
+
+from app.dependency.smtp_connection import get_smtp_connection
+from app.email_router.smtp_conn import SmtpConnection
+from app.util.logger import logger
+
+
+class EmailService:
+    def __init__(
+        self, smtp_conn: SmtpConnection = Depends(get_smtp_connection)
```

Comment:
Remove this unnecessary Depends, as the dependency injection method for email service inputs this explicitly.